committee/committee.go

```diff
@@ -227,9 +232,15 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 		for {
```

Comment:
should remove ec.terminate from SubscribeNewBlock()
let carrier use its own terminate channel